### ISSUE TITLE
How to use in Vue js & Vite?

### ISSUE THREAD

**Issue by liho00** [blikblum/wc-f7#29]

**Comment by blikblum** [comment]
Hi, this is just web components so should work in vite.

You should not use with vue. Better stick with direct f7 vue support

**Comment by liho00** [comment]
> Hi, this is just web components so should work in vite.
> 
> You should not use with vue. Better stick with direct f7 vue support

Nope, im not going to use framework 7 vue framework, because this framework fully depends on the f7-app components, i not able to use its components suchs as notification, toast and etc with my current project and other ui framework.

Besides, the most worst of framework 7 downside was its overide the <a href>, its basic component of web, hence i finding webcomponents of framework 7, I not yet try your example yet. Will try it later.

**Comment by blikblum** [comment]
Hi, this repository currently holds only a customized framework7 router that allows to load a web component instead of the f7 builtin component. So you can use a web component to handle things like state / events but each f7 ui component is defined as html markup (f7 core). Like in the examples: https://github.com/blikblum/wc-f7/blob/master/examples/starter-app/src/pages/form-page.js

There's no web component versions of each ui component.

I have some of those components (like smart-select or preloader) converted to web component in my projects but did not published
